### PR TITLE
HardwareMilestone

### PR DESCRIPTION
**Pull request by AndreiAndreescu**
### Author

Signed-off-by: Andreescu Andrei-Vlad <andrei.andreescu@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/andrei.andreescu/index.md.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/andrei.andreescu/index.md.md:85]: Please fix these links.

File: docs/project/2024/andrei.andreescu/index.md.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+[Light sensor and movement sensor](https://www.emag.ro/senzor-de-lumina-rgb-detector-de-gesturi-apds-9960-multicolor-apds9960-mod/pd/DLR798MBM/?utm_campaign=share%20product&utm_source=mobile%20app&utm_medium=ios) |[21 RON] |
+[Laser distance sensor](https://www.emag.ro/senzor-de-distanta-laser-tof-vl53l0x-aalbastru-vl53l0x-gy530-blue/pd/DQR798MBM/?utm_campaign=share%20product&utm_medium=ios&utm_source=mobile%20app)|[16 RON]|
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/andrei.andreescu/index.md.md:78]: Do you need these libraries?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by kaanmeister** [comment]
You have your schematic picture called `arhitecture.png` on your project documents but you wrote `architecture.png` on your index.md file to be able to upload picture. Change the picture name on your index.md file to be able to solve the build error.

**Comment by kaanmeister** [comment]
> You have your schematic picture called `arhitecture.png` on your project documents but you wrote `architecture.png` on your index.md file to be able to upload picture. Change the picture name on your index.md file to be able to solve the build error.

issuepullreq

For better understanding.

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/andrei.andreescu/kicad.jpg
Comment (alexandruradovici) [inline docs/project/2024/andrei.andreescu/kicad.jpg:1]: Export the schematics instead of the screenshot.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**